engine/env/spark/src/test/java/org/datacleaner/spark/JobResultPathTest.java

```diff
@@ -0,0 +1,152 @@
+package org.datacleaner.spark;
+
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+import junit.framework.TestCase;
+
+import org.apache.spark.SparkConf;
+import org.apache.spark.api.java.JavaSparkContext;
+import org.datacleaner.job.AnalysisJob;
+import org.junit.Test;
+
+/**
+ * This class tests the path of the result specified in the job properties file
+ * or otherwise. The default setting for 'fs.defaultFS' is 'file:///'
```

Comment:
How is the default setting "file:///" set? For me this is news, but also a bit of a surprise and strange thing in the test to rely on?